assets/lib/backbone/backbone.marionette.js

```diff
@@ -3010,7 +3010,7 @@
     // Internal method to ensure an `$childViewContainer` exists, for the
     // `attachHtml` method to use.
     getChildViewContainer: function(containerView, childView) {
-      if (!!containerView.$childViewContainer) {
+      if (!!containerView.$childViewContainer) { // childViewContainer was passed as an empty string.
```

Comment:
@StyleShit This lib only works for the container scenario if childViewContainer is an empty string for example:

`childViewContainer: '',` https://github.com/elementor/elementor/blob/master/assets/dev/js/editor/elements/views/container.js#L15

But it does not work if childViewContainer is a function that returns an empty string for example:

```
childViewContainer() {
    return '';
},
```

Because we have conditional logic we need `childViewContainer` as a function:

```
childViewContainer() {
	return 'boxed' === this.getContainer().settings.get( 'content_width' )
		? ' > .e-container__inner'
		: ''; // Child view of `content_width: full` is empty in order to use the parent element.
},
```

We also could not do the following because `this.getContainer()` would be undefined in this context:
(this.container, this.model is also undefined if we are not in a function).

```
childViewContainer: 'boxed' === this.getContainer().settings.get( 'content_width' )
		? ' > .e-container__inner'
		: '';
```

Reply:
@hein-obox We cannot touch Marionette.
Do whatever is needed to do to solve this problem. Consult with Tzvi or me if needed.

Reply:
@udidol Your suggestion worked.
Done.